src/app/lib/config/services/somali.js

```diff
@@ -250,10 +250,6 @@ export const service = {
         title: 'Barnaamijyada Idaacadda',
         url: '/somali/ka_qayb_gal/programmes',
       },
-      {
-        title: 'Sawirro',
-        url: '/somali/media/photogalleries',
-      },
     ],
```

Comment:
We also have to remove the audio from the Somali service `/somali/media/audio`

Reply:
🙈 Missed that. Fixed now.